### PR TITLE
fake JSON model of hello.wat

### PR DESCRIPTION
**Pull request by pavelsavara**
convert WIT from textual to JSON model form

### CODE REVIEW

**Comment by pavelsavara** [comment]
@AaronRobinsonMSFT FYI: @ilonatommy joined us to do this

File: hello/wat/hello.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            value: {
+                tag: 'ComponentTypeFunc',
+                value:
+                {
+                    params: [
+                        [
+                            "info",
+                            {
+                                tag: 'ComponentValTypePrimitive',
+                                value: PrimitiveValType.S32 // no guarantee that 4 is int32
```
Comment (pavelsavara) [inline hello/wat/hello.ts:166]: `s32` is `int32` s stands for signed

Comment (ilonatommy) [reply hello/wat/hello.ts:166]: We have "(func (param "info" 4))" and 4 can be whatever type, I guess it's int32 but there's no guarantee. That's what it means.

File: hello/wat/hello.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        {
+            tag: 'ComponentTypeDeclarationType',
+            value: {
+                tag: 'ComponentTypeDefined',
+                value: {
+                    tag: 'ComponentDefinedTypeRecord',
+                    members:
+                    [
+                        {
+                            name: "name",
```
Comment (pavelsavara) [inline hello/wat/hello.ts:65]: I updated the .wat and there are now 2 more members

File: hello/wat/hello.ts [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+            tag: 'ComponentTypeDeclarationType',
+            value: {
+                tag: 'ComponentTypeFunc',
+                value:
+                {
+                    params: [
+                        [
+                            "info",
+                            {
+                                tag: 'ComponentValTypePrimitive',
```
[the remark this answers is not in the document]
Comment (ilonatommy) [reply hello/wat/hello.ts:119]: this is helpful

**Pull request merged**